### ISSUE TITLE
R CMD check warning re: size

### ISSUE THREAD

**Issue by nsheff** [databio/GenomicDistributions#3]
we're including a few megs worth of data within the package. This leads to a warning because it wants `--resave-data`.

```
* checking data for ASCII and uncompressed saves ... WARNING
  
  Note: significantly better compression could be obtained
        by using R CMD build --resave-data
                        old_size new_size compress
  geneModels_hg19.RData    1.6Mb    948Kb       xz
  geneModels_hg38.RData    1.5Mb    940Kb       xz
  geneModels_mm10.RData    1.4Mb    882Kb       xz
  geneModels_mm9.RData     1.3Mb    816Kb       xz
  TSS_hg19.RData           256Kb    119Kb       xz
  TSS_hg38.RData           252Kb    116Kb       xz
  TSS_mm10.RData           237Kb    107Kb       xz
  TSS_mm9.RData            415Kb    171Kb       xz
```

another option is to switch to use that other Bioc package that had the metadata, which could be better.

**Comment by j-lawson** [comment]
It sounds like the data wasn't saved in the most efficient way. I've had to resave data before for Bioc packages to make it smaller: save(..., compress = "xz")

**Comment by nsheff** [comment]
I see, that's simpler than I was imagining. I should be able to do that.

ultimately I want to switch away from having these datasets built-in, though. But I should do this for now, at least.

**Comment by nsheff** [comment]
fixed in a4d36f8
